apps/studio/src/server/modules/site/site.types.ts

```diff
@@ -1,18 +1,12 @@
-export interface Sitemap {
-  parentTitle: string
-  childrenTitles: string[]
-  siblingTitles: string[]
-}
+import { type IsomerSiteProps } from '@opengovsg/isomer-components'
 
-export const SiteThemes = {
+export const SiteThemes: Record<string, IsomerSiteProps['theme']> = {
   classic: 'isomer-classic',
   next: 'isomer-next',
 } as const
 
-type SiteTheme = (typeof SiteThemes)[keyof typeof SiteThemes]
-
 export interface SiteConfig {
```

Comment:
suggestion(blocking): I think `SiteConfig` should derive its types directly from `IsomerSiteProps` using `Omit`. Otherwise it feels like we are duplicating the types definitions.

Reply:
fixed downstream
/

Reply:
per offline discussion - `SiteConfig` is site-wide config + footer/navbar. as such, just omitting the two should be ok